### ISSUE TITLE
Obtaining exports.csv file when using classify.py

### ISSUE THREAD

**Issue by lizard24** [ultralytics/yolov5#6286]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and [discussions](https://github.com/ultralytics/yolov5/discussions) and found no similar questions.

### Question

Hello,

firstly, many thanks for this excellent and easy-to-use framework!

I have started using the classifier branch. However, I noticed that only the weights are saved after training - no results.csv file or else as is the case for the main branch. Any chance this could be added?

Many thanks,

Lisa

**Comment by glenn-jocher** [comment]
@lizard24 yes this is a good point. The losses and metrics are different so they need to be logged a little differently for classification than detection. 

Another option is to simply add a logger to capture workspace outputs like this, though this encounters issues with colored and bold fonts:
```python
LOGGER.addHandler(logging.FileHandler('log.txt'))
```

**Comment by lizard24** [comment]
Great, that sounds like a good work-around! Many thanks for the quick reply!

**Comment by glenn-jocher** [comment]
@lizard24 I've opened up a PR for master that shows how this might work in https://github.com/ultralytics/yolov5/pull/6287

**Comment by lizard24** [comment]
I've implemented those changes but for some reason only get a log file in the main directory that only saves the first row of the output. Notably, I use Python 3.8 which prohibits me to ignore the errors in 'logging.FileHandler()'.

My inelegant workaround solution for now:

  
    def export_results(results):
    
        export, epoch = [], 1
    
        for item in results:
            if '100%' in item:
                _export = [ii for ii in item.split(' ') if ii!=''][:5]
                _export[0] = _export[0].split('/')[0]
                if _export[0]==str(epoch):
                    export.append(_export)
                    epoch+=1
            elif 'Training complete. Results saved to ' in item:
                save_dir = item.replace('Training complete. Results saved to ','')[:-1]
    
        with open('{}/output.txt'.format(save_dir), 'w') as f:
            for item in results:
                f.write(item+'\n')
    
        with open('{}/results.txt'.format(save_dir), 'w') as f:
            f.write('\t'.join(['epoch', 'gpu_mem', 'train_loss', 'val_loss', 'accuracy'])+'\n')
            for item in export:
                f.write('\t'.join(item)+'\n')
                
    output = !python classify.py
    
    export_results(output)

**Comment by glenn-jocher** [comment]
@lizard24 that's odd, the PR should work well in Linux and MacOS, but it's throwing errors on Windows. I just checked it now in Colab (python 3.7) and also see errors on the 'errors' argument. 

I'll try updating the PR to remove the argument.

**Comment by glenn-jocher** [comment]
@lizard24 after updating PR, Colab and Python 3.7 works now, but Windows still breaks. Results look like this:

<img width="1737" alt="Screen Shot 2022-01-17 at 2 39 20 PM" src="https://user-images.githubusercontent.com/26833433/149851933-d741da27-f6f5-4869-9208-3079adad8d47.png">
